### ISSUE TITLE
[Bug] alarm is set to turn off after one minute but doaent happend

### ISSUE THREAD

**Issue by lipici** [SimpleMobileTools/Simple-Clock#296]
[Bug] alarm is set to turn off after one minute but dosent happens. After one minute display goes black and you need to unlock phone to turn it off.

**Comment by satomst** [comment]
I have the same issue, v.5.6.

In my case, I want the (loud enough) timer sound to stop after 2-3 seconds, without me touching the phone. 

Changed the "Max timer duration" in Settings to 3 seconds, but the timer sound keeps looping unless I pull down the notification area (shows "Time expired") or Clock app's "refresh" button.

**Comment by ZarVladimirII** [comment]
A more crazy thing.
Until I updated from 5.6.0 to 5.7.0 the timer worked on my phone; counting down and turn automatically off. Everything was fine.
So I downgraded (needed to deinstall and reinstall) but now even with 5.6.0 it does not work anymore.
5.6.0 was first installed under Android 10. Both now are installed under Android 11.

(About abandon specific apps, SimpleMobileTools is a complete package, a universe)
